_data/toc.yaml

```diff
@@ -1531,6 +1531,8 @@ manuals:
     section:
       - path: /docker-hub/publish/
         title: Overview
+      - path: /docker-hub/publish/insights-analytics/
+        title: Insights & analytics
```

Comment:
Unrelated change? (Looks like this was not meant to be part of this PR)

Reply:
Removed.